### DISCUSSION TITLE
As a user on desktop using the installed Tidal App I have the ability to display a mini player in the corner of the window with previous,next, pause/play controls

### DISCUSSION THREAD

**Discussion by jr-tkinne** [tidal-music/discussions#48] [Ideas & Feature requests]
It'd be nice to have the ability to have a mini-player that can be docked in the bottom right of the window with controls from the Tidal App

**Comment by stefanmoro** [comment]
Hey @thomaskinne-jr,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API, open source projects etc).

As this question relates to an existing product, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
